### PR TITLE
Feature/go to associated node link

### PR DESCRIPTION
**Pull request by slakh96**
Completes https://github.com/orgs/VerWiki/projects/1#card-52007354 and adds a ton of style improvements

### CODE REVIEW

File: frontend/src/components/InfoWindow/InfoWindow.jsx [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import React from "react";
 import DOMPurify from "dompurify";
+import { Button, BackTop } from "antd";
```
Comment (mit1221) [inline frontend/src/components/InfoWindow/InfoWindow.jsx:3]: I think we should stay consistent with the UI library we use throughout the app. Currently, Material UI is being used for everything else.

**Comment by mit1221** [comment]
There's a small bug with the info window. The first time you right click a node, the info window opens with an animation. However, if you then close this window, and then open it again by right clicking a node, there is no animation.

File: frontend/src/pages/CoursePage/CoursePage.jsx [hunk 2]
```diff
@@ -31,14 +29,15 @@ export const CoursePage = () => {
...
-      <h1 className={styles.heading}>{courseData[courseId].name}</h1>
-      <div>{Viewer}</div>
-    </>
-  );
+  return <div>{Viewer}</div>;
 };
+
+/**
+ * TODO:
+ * 1. put course name and explore button on the toolbar
```
Comment (mit1221) [inline frontend/src/pages/CoursePage/CoursePage.jsx]: Done

File: frontend/src/components/InfoWindow/InfoWindow.jsx (hunk 1 above)
Comment (slakh96) [reply frontend/src/components/InfoWindow/InfoWindow.jsx:3]: @sultansidhu what do you think?

File: frontend/src/pages/CoursePage/CoursePage.jsx (hunk 2 above)
Comment (slakh96) [reply frontend/src/pages/CoursePage/CoursePage.jsx]: Removed

File: frontend/src/components/InfoWindow/InfoWindow.jsx (hunk 1 above)
Comment (sultansidhu) [reply frontend/src/components/InfoWindow/InfoWindow.jsx:3]: Hmm I think we might wanna focus more on preserving the aesthetic of the overall project? For example, Antd has some gorgeous components that fit very well with material UI (even better than their material UI counterparts I'd argue). What do you guys think @slakh96 @mit1221

Comment (mit1221) [reply frontend/src/components/InfoWindow/InfoWindow.jsx:3]: I see. Ya, we can just leave it the way it is for now then. Maybe we can move everything to Antd later but that's definitely not a priority.

**Review by mit1221 — APPROVED** [review]

**Pull request merged**